src/mathui.js

```diff
@@ -96,6 +97,10 @@ export default class MathUI extends Plugin {
 			this._closeFormView();
 		} );
 
+		this.listenTo( formView, 'mathlive:virtualKeyboard:toggle', ( event, state ) => {
```

Comment:
look at https://github.com/bethinkpl/ckeditor5-bethink/blob/main/src/plugin-bethink-lucidcharts.js#L21
maybe we can delegate events in the same way but into the editor directly

Reply:
I tried this, but I would have to pass an instance of the editor or pluins down to MathLiveView and it ended up with circural dependencies error